src/Finder.php

```diff
@@ -27,6 +27,10 @@ public function __construct()
         $this
             ->files()
             ->name('*.php')
+            ->ignoreDotFiles(false)
+            ->notPath('~(^|/)\.(?!php_cs(\.dist)?(/|$)).+(/|$)~')
```

Comment:
comment why this is ignored would be good

Reply:
This equals to original `ignoreDotFiles(true)`, copied from Symfony Finder.